### PR TITLE
VAOS update contact info page

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
This adds email to the contact info page, according to our new mocks.

This also addresses https://github.com/department-of-veterans-affairs/va.gov-team/issues/2862 and https://github.com/department-of-veterans-affairs/va.gov-team/issues/2846.

## Testing done
Local and unit testing

## Screenshots
Screen Shot 2019-10-31 at 5 13 59 PM

## Acceptance criteria
- [ ] Contact page has email, updated copy

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by narin — APPROVED** [review]
LGTM.  Failing test seems unrelated to this body of work.

**Review by rianfowler — APPROVED** [review]

File: src/applications/vaos/containers/ContactInfoPage.jsx [hunk 1]
```diff
@@ -34,35 +34,57 @@ const initialSchema = {
...
     },
     afternoon: {
-      'ui:title': 'Afternoon (noon - 4 p.m. EST)',
+      'ui:title': 'Afternoon (noon - 4 p.m.)',
       'ui:options': {
         widgetClassNames: 'vaos-form__checkbox',
       },
     },
     evening: {
-      'ui:title': 'Evening (4 p.m. to 8 p.m. EST)',
```
Comment (cvalarida) [inline src/applications/vaos/containers/ContactInfoPage.jsx]: Interesting. Is it communicated that all times are eastern? Or are they?

File: src/applications/vaos/containers/ContactInfoPage.jsx [hunk 2]
```diff
@@ -84,22 +106,26 @@ export class ContactInfoPage extends React.Component {
...
-      >
-        <FormButtons
-          onBack={this.goBack}
-          pageChangeInProgress={pageChangeInProgress}
-        />
-      </SchemaForm>
+      <div>
+        <h1 className="vads-u-font-size--h2">Contact information</h1>
+
+        <SchemaForm
```
Comment (cvalarida) [inline src/applications/vaos/containers/ContactInfoPage.jsx]: Huh. I think GH doesn't like the React fragment syntax; the syntax highlighting is all wonky now. :man_shrugging: 

Then again, it looks like it was kinda weird before this anyhow...Ah, well.

**Review by cvalarida — APPROVED** [review]

File: src/applications/vaos/containers/ContactInfoPage.jsx (hunk 1 above)
Comment (jbalboni) [reply src/applications/vaos/containers/ContactInfoPage.jsx]: They're local times, but we're supposed to show the timezone. I'm going to leave that to a later ticket and am making this change so it isn't misleading in the meantime.

Comment (jbalboni) [reply src/applications/vaos/containers/ContactInfoPage.jsx]: They're local times, but we're supposed to show the timezone. I'm going to leave that to a later ticket and am making this change so it isn't misleading in the meantime.

**Pull request merged**